ballerina/tests/from_json_test.bal

```diff
@@ -1314,18 +1323,6 @@ isolated function testParseAsTypeNegative6() {
     test:assertEquals((<Error>x).message(), "incompatible value '4' for type 'int' in field 'house'");
 }
 
-@test:Config
```

Comment:
Why is this test removed?

Reply:
This is not a duplicate field test. The earlier error we encountered was incorrect